src/dfx/src/util/clap/install_mode.rs

```diff
@@ -0,0 +1,80 @@
+use anyhow::bail;
+use clap::Args;
+use ic_utils::interfaces::management_canister::builders::{
+    CanisterUpgradeOptions, InstallMode, WasmMemoryPersistence,
+};
+
+use crate::lib::error::DfxResult;
+
+/// CLI options for the mode of installing canister.
```

Comment:
```suggestion
/// CLI options for the mode of canister installation.
```
I like your formulation below better